frontend/common/src/components/inputPartials/Fieldset/Fieldset.tsx

```diff
@@ -7,7 +7,7 @@ import InputError from "../InputError/InputError";
 
 export interface FieldsetProps {
   /** The text for the legend element. */
-  legend: string;
+  legend?: string;
```

Comment:
I think providing a way for the legend to be hidden would be more appropriate. A `<fieldset>` must have a `<legend>` as the first child.

"The first element inside the fieldset must be a legend element, which provides a label or description for the group."

REF: https://www.w3.org/WAI/WCAG21/Techniques/html/H71.html

Reply:
Good point!  I've made it mandatory again and added a `hideLegend` tag in the same way that `hideOptional` exists.

https://github.com/GCTC-NTGC/gc-digital-talent/pull/3353/commits/c59416a43f884ef93731f1f6c63dc165e80f0201